src/components/ScreeningsSection.vue

```diff
@@ -2,17 +2,19 @@
   <div class="screening-section">
     <h2 class="screening-section__title">Screenings:</h2>
     <p class="screening-section__content">Friday 11/02/2022</p>
-    <div>
+    <div class="screening-section__input-wrapper">
       <DayTabsList></DayTabsList>
+      <MovieSelect></MovieSelect>
```

Comment:
Just a tip: [In Vue, if your component has no children, you can make it self-closing](https://v2.vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended)